Website/Website/Form/VisitorOverview.aspx

```diff
@@ -4,30 +4,74 @@
+    <link href="~/Resources/css/style.css" rel="stylesheet" type="text/css" />
+    <link href="~/Resources/css/style_visitor.css" rel="stylesheet" type="text/css" />
 </head>
 <body>
-    <form id="form1" runat="server">
-        <div>
-            <h1>
-                <asp:Label ID="lblVisitor" runat="server" Text="Welkom "></asp:Label>
-            </h1>
-            <p>Hier ziet u een overzicht van uw festivals.</p>
-
-        </div>
-        <asp:Button ID="btnAddTicket" runat="server" Text="Ticket toevoegen"/>
-        <asp:Table ID="tblTickets" runat="server" CellPadding="5" CellSpacing="0" GridLines="Both">
-            <asp:TableRow runat="server">
-                <asp:TableCell runat="server"><b>Festival</b></asp:TableCell>
-                <asp:TableCell runat="server"><b>Ticket soort</b></asp:TableCell>
-                <asp:TableCell runat="server"><b>Datum</b></asp:TableCell>
-                <asp:TableCell runat="server"><b>Programma</b></asp:TableCell>
-            </asp:TableRow>
-        </asp:Table>
-        <asp:Button ID="btnLogout" runat="server" Height="30px" OnClick="btnLogoutClick" Text="Uitloggen" Width="108px" />
-        <p>
-            <asp:Button ID="btnGenre" runat="server" Height="30px" Text="Mijn genres" Width="107px" />
-        </p>
+    <form id="formBezoekerHoofdscherm" runat="server">
+
+        <section class="container-fluid">
+
+            <div class="row extra-padding-top-xs">
+                <div class="col-9"></div>
+                <div class="col-3 text-right">
+                    <asp:Button ID="btnLogout" runat="server" OnClick="btnLogoutClick" Text="Uitloggen" CssClass="btn btn-primary btn-primary-small" />
+                </div>
+            </div>
+
+            <div class="clearfix"></div>
+
+            <div class="row extra-padding-top-xs">
+                <div class="col-3"></div>
+                <div class="col-6 text-center">
+                    <h1 style="padding: 0;">
+                        <asp:Label ID="lblVisitor" runat="server" Text="Welkom "></asp:Label></h1>
+                </div>
+                <div class="col-3 text-right">
+                    <asp:Button ID="btnGenre" runat="server" Text="Mijn genres" CssClass="btn btn-primary btn-primary-small" />
+                </div>
+            </div>
+
+        </section>
+
+        <section class="container text-center">
+
+            <h3 class="extra-padding-top-sm">Hier ziet u een overzicht van uw festivals.</h3>
+
+            <div class="extra-padding-top-sm">
+                <asp:Button ID="btnAddTicket" runat="server" Text="Ticket Toevoegen" CssClass="btn btn-primary btn-primary-big" />
+            </div>
+
+            <div class="extra-padding-top-sm tabel-wrapper">
+                <asp:Panel ID="pnlTickets" runat="server">
+                    <div class="row tabel">
+                        <div class="col tabel-center-verticaal">
```

Comment:
de meeste css classes zijn in t engels nu (komt door bootstrap) maar misschien is t handiger als we alle css classes dan in t engels doen

Reply:
let ook op dat je dat in de .cs bestand ook aanpast